optimum/exporters/neuron/convert.py

```diff
@@ -458,27 +458,32 @@ def export_neuronx(
 
 
 def add_stable_diffusion_compiler_args(config, compiler_args):
-    if hasattr(config._config, "_name_or_path"):
-        sd_components = ["text_encoder", "vae", "vae_encoder", "vae_decoder"]
-        if any(component in config._config._name_or_path.lower() for component in sd_components):
+    identifier = (
+        getattr(config._config, "_name_or_path", "").lower() + " " + getattr(config._config, "_class_name", "").lower()
+    )
```

Comment:
nit: I find it hard to read.
```suggestion
    identifier = getattr(config._config, "_name_or_path", "") + " " + getattr(config._config, "_class_name", "")
    identifier = identifier.lower()
```